### ISSUE TITLE
Report the effect for every genes 

### ISSUE THREAD

**Issue by smw1414** [charite/jannovar#530]
**Is your feature request related to a problem? Please describe.**
Jannovar picks transcript by the most severe consequences of transcript. But there are genes that overlapped with each other and with the same consequences. If report only one gene of information may lost the very critical information . For example, [X: 101399747](https://www.ncbi.nlm.nih.gov/clinvar/variation/10768/) is a very important position for several diseases. But Jannovar reports RPL36A-HNRNPH2 instead of the disease-causative gene GLA (vcf mode).   

**Describe the solution you'd like**
I know that the --show-all could solve this problem, but the data size become very big. So I would recommend that report variants by gene.    
Assume GeneA  and GeneB are overlapped with each other and Transcript1, Transcript2 and Transcript3 have the same consequences. 
Jannovar may just random pick one of transcript(I guess). I recommend to report (Transcript1 or Transcirpt2) and (Transcript3). Then, the information of GeneA and GeneB will be reported, respectively. 
GeneA Transcript1 
GeneA Transcript2
GeneB Transcript3

**Describe alternatives you've considered**
N/A

**Additional context**
Example of genes that overlapped with each other.   
![圖片](https://user-images.githubusercontent.com/28624336/123538525-e8118e00-d767-11eb-9205-f06983d8f360.png)

**Comment by holtgrewe** [comment]
I agree, this should be implemented and is the same as in [varfish-annotator](https://github.com/bihealth/varfish-annotator).
